drivers/ipm/ipm_nrfx_ipc.c

```diff
@@ -36,7 +37,8 @@ static void nrfx_ipc_handler(uint32_t event_mask, void *p_context)
 			__ASSERT(event_idx < NRFX_IPC_ID_MAX_VALUE,
 				 "Illegal event_idx: %d", event_idx);
 			event_mask &= ~BIT(event_idx);
-			nrfx_ipm_data.callback(nrfx_ipm_data.callback_ctx,
+			nrfx_ipm_data.callback(nrfx_ipm_data.dev,
```

Comment:
This is a single instance part of the driver, so you could use `DEVICE_GET(ipm_nrf)` here and avoid adding the `dev` field.

Reply:
ok